src/jimgw/transforms.py

```diff
@@ -368,6 +380,111 @@ def __init__(
         }
 
 
+@jaxtyped(typechecker=typechecker)
```

Comment:
I am wondering for single event specific transform, perhaps we should move these into the single_event directory